Doc/whatsnew/3.13.rst

```diff
@@ -122,11 +122,22 @@ Optimizations
 Deprecated
 ==========
 
+Pending Removal in Python 3.15
+------------------------------
+
 * :mod:`wave`: Deprecate the ``getmark()``, ``setmark()`` and ``getmarkers()``
   methods of the :class:`wave.Wave_read` and :class:`wave.Wave_write` classes.
   They will be removed in Python 3.15.
```

Comment:
This sentence now sounds redundant with the section title. Same remark with "will be removed in Python 3.16" below.

Reply:
I'll instead remove these section titles, I just realised they're missing this intro text (emphasis added):

> The following Python APIs have been _deprecated in earlier Python releases_, and will be removed in Python 3.12.

For example:

* https://docs.python.org/3.13/whatsnew/3.11.html#pending-removal-in-python-3-12
* https://docs.python.org/3.13/whatsnew/3.12.html#pending-removal-in-python-3-13
* https://docs.python.org/3.13/whatsnew/3.12.html#pending-removal-in-future-versions